app/lib/event_bus.rb

```diff
@@ -85,3 +85,5 @@ def self.reset!
 EventBus.subscribe :filter_created, ->(filter) { AuditLog::FilterCreated.create_audit_record(filter) }
 EventBus.subscribe :filter_updated, ->(filter) { AuditLog::FilterUpdated.create_audit_record(filter) }
 EventBus.subscribe :filter_destroyed, ->(filter) { AuditLog::FilterDestroyed.create_audit_record(filter) }
+EventBus.subscribe :box_sync_invoked, ->(box) { AuditLog::BoxSyncInvoked.create_audit_record(box) }
+EventBus.subscribe :box_sync_all_invoked, -> { AuditLog::BoxSyncAllInvoked.create_audit_record }
```

Comment:
`BoxSyncAllInvoked` znie naozaj divne. Potrebujeme vlastne rozlisovat medzi dvoma eventami?

Reply:
Ja neviem, ci potrebujeme. Je to technicky / admin log, tak by som cakal, ze tam taka "presnost" nevadi. Ale mozme aj zjednotit.